File: go/data/gorm/model/address.go
package model

import (
	"gorm.io/gorm"
	"time"
)

type Address struct {
	//gorm.Model -- overriding this with custom model for primary key
	Id          string         `gorm:"primary_key;type:text;default:gen_random_uuid()" json:"id"`
	CreatedAt   time.Time      `json:"createdAt"`
	UpdatedAt   time.Time      `json:"updatedAt"`
	DeletedAt   gorm.DeletedAt `gorm:"index" json:"deletedAt"`
	FirstName   string         `json:"firstName"`
	LastName    string         `json:"lastName"`
	CompanyName string         `json:"companyName"`
	Address     string         `json:"address"`
	City        string         `json:"city"`
	County      string         `json:"county"`
	State       string         `json:"state"`
	Zip         string         `json:"zip"`
	Phone1      string         `json:"phone1"`
	Phone2      string         `json:"phone2"`
	Email       string         `json:"email"`
	Web         string         `json:"web"`
}

File: rust/data/diesel/model/src/address.rs
use diesel::prelude::*;

#[derive(Queryable)]
pub struct Address {
    pub id: String,
    // pub created_at: PgTimestamp,
    // pub updated_at: PgTimestamp,
    // pub deleted_at: PgTimestamp,
    pub first_name: String,
    pub last_name: String,
    pub company_name: String,
    pub address: String,
    pub city: String,
    pub county: String,
    pub state: String,
    pub zip: String,
    pub phone_1: String,
    pub phone_2: String,
    pub email: String,
    pub web: String,
}